spectec/spec/6-reduction.watsup

```diff
@@ -305,6 +305,9 @@ rule Step/memory.grow-succeed:
 rule Step/memory.grow-fail:
   z; (CONST I32 n) (MEMORY.GROW)  ~>  z; (CONST I32 $(-1))
 
+rule Step_read/memory.size:
```

Comment:
Nit: can you move this before the memory.grow rules, just to have the same order as for the table instructions?

Reply:
Sure!